src/encoded/tests/data/inserts/treatment.json

```diff
@@ -169,5 +169,15 @@
         "treatment_term_id": "CL:0000815",
         "treatment_term_name": "regulatory T cell",
         "treatment_type": "injection"
+    },
+        {
```

Comment:
It looks like this curly bracket is indented an extra tab inward

Reply:
Fixed